app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -257,6 +258,10 @@ public void onResume() {
                 }
             }
             intent.removeExtra(BundleKeys.BUNDLE_KEY_SESSION_ALARM_SESSION_ID); // jump to given sessionId only once
+            int notificationId = intent.getIntExtra(BundleKeys.BUNDLE_KEY_SESSION_ALARM_NOTIFICATION_ID, INVALID_NOTIFICATION_ID);
+            if (notificationId != INVALID_NOTIFICATION_ID) {
+                appRepository.deleteNotificationId(notificationId);
```

Comment:
This performs disk I/O on the main thread. There's also no reason to run this every time the activity is resumed. 
Trigger the action in `onCreate()` and run the operation in a background thread?

Reply:
I added this in `onResume()` to remove the database entry when the user tapped the notification.

Not sure if I want to target the main thread issue here since it is a general issue with the `AppRepository`.

Reply:
:heavy_check_mark: Thanks. I moved the code to `MainActivity` in a4f7c3a1d247992e2f684f6b1a1c7885d4c49a1e.

I also clarified a few method names and KDoc.